specification/index.bs

```diff
@@ -154,6 +155,8 @@ Filenames beginning with an underscore (`_`) are reserved for use by user agent.
 
 # Isolated worlds
 
+<dfn>Worlds</dfn> are isolated JavaScript contexts with access to the same underlying DOM tree but their own set of wrappers around those DOM objects.
```

Comment:
maybe referece the fact that the entire variable space is isolated?  (It's not just the DOM objects)